css-sass/index.html

```diff
@@ -0,0 +1,321 @@
+<!DOCTYPE html>
+<html lang="en-us">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>index</title>
+  <link rel="stylesheet" href="./styles/css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+
+  <section class="flex justify-between items-center mt-3 w-80 m-inline">
+    <div class="flex items-center gap-1">
+      <img src="../assets/svg/logo_planIT.svg" class="w-3 h-4">
+      <img src="../assets/svg/planit_letters.svg" class="w-5 h-1">
+    </div>
+
+    <ul class="list-none flex gap-3">
+      <li><a class="text-black no-underline font-light text-base" href="#Caracteristicas">Características</a></li>
+      <li><a class="text-black no-underline font-light text-base" href="#Precio">Precio</a></li>
+      <li><a class="text-black no-underline font-light text-base" href="#Contacto">Contacto</a></li>
+    </ul>
+    <div class="flex justify-around gap-3 items-center">
+      <a class="text-black no-underline font-light text-base" href="#login">Ingresar</a>
+      <button type="button"
+        class="py-0-8 px-4 rounded-lg bg-white text-base font-medium cursor-pointer tracking-tight leading-tight">Registrarse
+        gratis</button>
+    </div>
+  </section>
+
+  <section class="flex justify-between mt-4 gap-5 w-80 m-inline">
+    <div class="flex flex-col justify-center justify-between">
+      <header class="flex flex-col  items-start gap-2">
+        <div class="flex items-center gap-1">
+          <img src="../assets/svg/p_planit.svg" alt="logo" class="w-1 h-1">
+          <label class="text-base">Empresas</label>
+        </div>
+        <h2 class="font-semibold	 text-3xl tracking-tight leading-tight text-transparent-bgclip">
+          <span class="text-fill-black ">Generá acciones que</span> sorprendan a tu equipo y clientes
+        </h2>
+      </header>
+      <div class="flex gap-2 mt-2">
+        <button type="button"
+          class="py-0-9 px-5 rounded-lg border-none bg-primary-gradient text-white shadow-md cursor-pointer">REGISTRATE
+          ACÁ</button>
```

Comment:
We usually use css for applying uppercase to a text. In this link you can find some guidelines on this topic: https://www.freecodecamp.org/news/all-caps-in-css-how-to-uppercase-text-with-style/#:~:text=If%20you%20want%20to%20have,can%20hardcode%20that%20in%20HTML.